src/Symfony/Component/Form/Test/TypeTestCase.php

```diff
@@ -34,6 +36,20 @@ protected function setUp()
         $this->builder = new FormBuilder(null, null, $this->dispatcher, $this->factory);
     }
 
+    protected function getTypedExtensions()
```

Comment:
Thanks, it was supposed to be `getTypeExtensions`, I don't know how that d sneaked in there